### PR TITLE
opacity setting UI

### LINKED ISSUE #768: Opacity Feature UI @Sketch Page
## Sample Snapshot:

SKetch_opacoty

## Description:

Make the opacity option more appealing (i.e improve the UI of the slider). Additionally, remove the ability to see the amount of opacity. Having  both the quantitative option as well as GUI based option doesn't make much sense

### PR DESCRIPTION
**Pull request by ashuvssut**
closes #768

when not hovered
image

when hovered
image

----------------------------------
_- I have participated in GSSOC and CrossWoC_

### CODE REVIEW

**Comment by smaranjitghose** [comment]
Good job @ashuvssut.

For the sake of fairness, from next time just remember to comment on the issue before raising the PR☺

File: src/pages/Sketch/components/Canvas.module.css [hunk 1]
```diff
@@ -49,9 +49,9 @@
 
 /* ----- Mouse Position ----- */
 .mousePosition {
-  position: absolute;
-  bottom: 11px;
-  right: 30px;
+  position: fixed;
```
Comment (smaranjitghose) [inline src/pages/Sketch/components/Canvas.module.css:52]: Convert this to scss please

**Review by smaranjitghose — APPROVED** [review]

File: src/pages/Sketch/components/Canvas.module.css (hunk 1 above)
Comment (ashuvssut) [reply src/pages/Sketch/components/Canvas.module.css:52]: @smaranjitghose As you asked me before to open issues to convert css to scss... I will open those just right after my current assigned issues get completed. I am just facing a little trouble with some issues thats why its taking time

I will complete all my issue within one day

I will open css to scss issues just right after I finish my assigned work

Comment (smaranjitghose) [reply src/pages/Sketch/components/Canvas.module.css:52]: Sure!

**Pull request merged**